content/sensu-go/6.1/web-ui/search.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Search in the web UI"
+linkTitle: "Search in the Web UI"
+description: "The Sensu web UI supports searching the Events, Entities, Silences, Checks, Handlers, Filters, and Mutators pages. Learn more about searches in the Sensu web UI."
+weight: 30
+version: "6.1"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-6.1:
+    parent: web-ui
+---
+
+The Sensu web UI includes basic search functions you can use to build customized views of your Sensu resources.
+Sensu also supports advanced web UI searches based on a wider range of resource attributes and custom labels as a [commercial feature][1].
+
+When you apply a search to a web UI page, it creates a unique link for the page of search results.
+You can bookmark these links and share your favorite search combinations.
+You can also [save your favorite searches][8].
+
+## Search operators
+
+Web UI search supports two equality-based operators, two set-based operators, one substring matching operator, and one logical operator.
+
+| operator  | description        | example                |
+| --------- | ------------------ | ---------------------- |
+| `==`      | Equality           | `check.publish == true`
+| `!=`      | Inequality         | `check.namespace != "default"`
+| `in`      | Included in        | `linux in check.subscriptions`
+| `notin`   | Not included in    | `slack notin check.handlers`
+| `matches` | Substring matching | `check.name matches "linux-"`
+| `&&`      | Logical AND        | `check.publish == true && slack in check.handlers`
+
+For details about operators, see [API response filtering operators][7].
+
+## Use quick search
+
+The web UI quick search allows you to query Sensu resources without using search syntax.
+Type your search term into the search field on any page of the web UI and press `Enter`.
+Sensu will auto-complete a simple search statement for the resources on that page using substring matching.
+
+For example, on the Events page in the web UI, if you type `mysql` into the search field, Sensu will auto-complete the search statement to `event.check.name matches "mysql"`.
+
+## Create basic searches
+
+Sensu includes these basic search functions:
+
+- **Events page**: search by entity, check, status, and silenced/unsilenced.
+- **Entities page**: search by entity class and subscription.
+- **Silences page**: search by check and subscription.
+- **Checks page**: search by subscription and published/unpublished.
+- **Handlers page**: search by handler type.
+- **Filters page**: search by action.
+
+If you are using the [basic web UI search functions][5], you can create a search by clicking in the search bar at the top of the web UI page:
+
```

Comment:
I need to understand this better -- there are many more search functions listed when I just click in the search bar, but I think I'm using a licensed version. Are these additional search functions turned off at 100 entities, or are they available without a license too?

Reply:
Cameron suggests a single commercial note on the page that explains this page describes the web UI that's available in our commercial distribution. If you compile from OSS, you may not see all of these features.

It's also possible that a note is not necessary, or that this information should be included at https://docs.sensu.io/sensu-go/latest/platforms/#build-from-source instead.